src/Pages/Apartment/components/PriceChart.js

```diff
@@ -0,0 +1,221 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import Highcharts from 'highcharts';
+import HighchartsReact from 'highcharts-react-official';
+
+export default function PriceChart(props) {
+  const tabbedData = props.graphData[props.activeTab];
+  const sellingApartmentData = tabbedData && tabbedData.trade_apartments_data;
+  const rentApartmentData = tabbedData && tabbedData.rent_apartments_data;
+  const years = [];
+
+  let testValue = [];
+  let testDate = [];
+  let testValueForRent = [];
+  let testDateForRent = [];
+  let sellingYears = [];
+  let rentingYears = [];
+  let sellingDataForX = [];
+  let sellingDateForX = [];
+  let rentingDataForX = [];
+  let rentingDateForX = [];
+
+  for (let idx in tabbedData && tabbedData.trade_apartments_data) {
+    for (let year in tabbedData && tabbedData.trade_apartments_data[idx]) {
+      years.push(year);
+      sellingYears.push(Object.values(tabbedData.trade_apartments_data[idx]));
+    }
+  }
```

Comment:
반복되는 부분 변수로 설정해서 사용하는게 간결할 것 같습니다 

```suggestion
let aptData = tabbedData.trade_apartments_data;

  for (let idx in tabbedData && aptData ) {
    for (let year in tabbedData && aptData[idx]) {
      years.push(year);
      sellingYears.push(Object.values(aptData[idx]));
    }
  }
```

Reply:
아 훨씬 보기 좋네요!! 수정했습니다!